shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/migration/general/MySQLMigrationGeneralIT.java

```diff
@@ -86,13 +86,13 @@ public void assertMigrationSuccess() throws SQLException, InterruptedException {
         JdbcTemplate jdbcTemplate = new JdbcTemplate(getSourceDataSource());
         Pair<List<Object[]>, List<Object[]>> dataPair = ScalingCaseHelper.generateFullInsertData(keyGenerateAlgorithm, parameterized.getDatabaseType(), 3000);
         log.info("init data begin: {}", LocalDateTime.now());
-        jdbcTemplate.batchUpdate(getExtraSQLCommand().getFullInsertOrder(), dataPair.getLeft());
+        jdbcTemplate.batchUpdate(String.format(getExtraSQLCommand().getFullInsertOrder(), SOURCE_TABLE_ORDER), dataPair.getLeft());
```

Comment:
Could we add parameter for `getExtraSQLCommand().getFullInsertOrder()`, then IT case could just get whole SQL